taskPage1.css

```diff
@@ -0,0 +1,80 @@
+.pending-task {
+    height: 80px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5693010)